src/processLambda/app/app.py

```diff
@@ -1,68 +1,120 @@
+    return attendees['Items'][0]['AttendeeInfo']
+
+def process_files(objs_keys, MEETING_ID, file_type, *attendee):
+    # print(objs_keys)
+    if attendee:
+        attendeeStr = "-" + attendee[0]
+    else:
+        attendeeStr = ""
+        
+    with open('/tmp/' + file_type +attendeeStr+'_list.txt', 'w') as f:
+        for k in objs_keys:
+            basename = os.path.splitext(k)[0]
+            # print(basename)
+            ffmpeg_cmd = "ffmpeg -i /tmp/" + k + " -bsf:v h264_mp4toannexb -f mpegts -framerate 15 -c copy /tmp/" + basename + attendeeStr + "-" + file_type + ".ts -y"
+            command1 = shlex.split(ffmpeg_cmd)
+            # print (command1)
+            p1 = subprocess.run(command1, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+            # print(p1)
+            f.write(f'file \'/tmp/{basename}{attendeeStr}-{file_type}.ts\'\n')
+
+    ffmpeg_cmd = "ffmpeg -f concat -safe 0 -i /tmp/" + file_type + attendeeStr + "_list.txt  -c copy /tmp/"+file_type+attendeeStr+".mp4 -y"
+    command1 = shlex.split(ffmpeg_cmd)
+    p1 = subprocess.run(command1, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+    print(str(p1))
+    s3.upload_file('/tmp/'+file_type+attendeeStr+'.mp4', SOURCE_BUCKET, "captures/" + MEETING_ID + "/processed" + '/processed-'+file_type+attendeeStr+'.mp4')
+    processedUrl = s3.generate_presigned_url('get_object', Params={'Bucket': SOURCE_BUCKET, 'Key': "captures/" + MEETING_ID + "/processed" + '/processed'+attendeeStr+"-"+file_type+'.mp4' })
+    
+    return processedUrl
+    
+    
 def handler(event, context):
     print(event)
-    time.sleep(15)
-    meetingBody =  json.loads(event['body'])
-    MEETING_ID = meetingBody['meetingId']
-    prefix = SOURCE_PREFIX + '/' + MEETING_ID + '/audio'
-    client = boto3.client('s3')
+    MEETING_ID = event.get('detail').get('meetingId')
+    print(MEETING_ID)
 
-    response = client.list_objects_v2(
+    audioPrefix = SOURCE_PREFIX + '/' + MEETING_ID + '/audio'
+    videoPrefix = SOURCE_PREFIX + '/' + MEETING_ID + '/video'
+    
+    audioList = s3.list_objects_v2(
         Bucket=SOURCE_BUCKET,
         Delimiter='string',
         MaxKeys=1000,
-        Prefix=prefix
+        Prefix=audioPrefix
     )
-    objects = response.get('Contents', [])
+    audioObjects = audioList.get('Contents', [])
+    print(audioObjects)
+    
+    videoList = s3.list_objects_v2(
+        Bucket=SOURCE_BUCKET,
+        Delimiter='string',
+        MaxKeys=1000,
+        Prefix=videoPrefix
+    )
+    videoObjects = videoList.get('Contents', [])
+    print(videoObjects)
+    
+    if videoObjects:
+        file_list=[]
+        file_type = 'video'
+        for object in videoObjects:
+            path, filename = os.path.split(object['Key'])
+            s3.download_file(SOURCE_BUCKET, object['Key'], '/tmp/' + filename)
+            # print(filename)
+            file_list.append(filename)
+    
+        objs_keys = list(filter(lambda x : 'mp4' in x, file_list))
```

Comment:
You could use a list comprehension here which is more pythonic: `objs_keys = [file for file in file_list if file.endswith('.mp4')]`